.gitmodules

```diff
@@ -1,6 +0,0 @@
-[submodule "lib/websocketpp"]
```

Comment:
Please remove all changes that are not related to hunterization:
- https://docs.hunter.sh/en/latest/creating-new/patch.html

```
Please do push commits only related to hunterization
```

Reply:
well, this is sorta related cause the submodules are no longer required so no point including them?

Reply:
No need for hard remove, just don't use it in CMakeLists.txt. It's a source of conflict for future versions.